web_api/core/models.py

```diff
@@ -210,6 +226,19 @@ class Account(BaseModel):
         help_text="GitHub username for account with installation.",
     )
     github_account_type = models.CharField(max_length=255, choices=AccountType.choices)
+    trial_expiration = models.DateTimeField(null=True)
+    trial_start = models.DateTimeField(null=True)
+    trial_started_by = models.ForeignKey(
+        User, null=True, on_delete=models.SET_NULL, related_name="+"
+    )
+    trial_email = models.CharField(blank=True, max_length=255)
```

Comment:
We're doing trials outside of Stripe. No credit card required. I plan to manually send trial reminder emails for the time being.